packages/react-slider/src/utils/getMarkPercent.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Gets the current percentage position for the marks with respect to adjacent marks.
+ * This is used primarily for positioning with CSS grid.
+ *
+ * @param markValues The marks percentage position relative to their individual positions.
```

Comment:
Maybe this should be obvious but it's not quite clear to me what "relative to their individual positions" means

Reply:
There may be better wording for this. I'll see if I can improve it and add an example in the follow up pr. Here is an example of the difference. 

#### markValues
```jsx
Rail: width = 100px
Marks: [0, 25, 50, 75, 100]
getMarkPercent: 0%, 25%, 50%, 75%, 100%
```

#### getMarkPercent
```jsx
Rail: width = 100px
Marks: [0, 25, 50, 75, 100]
getMarkPercent: 0%, 25%, 25%, 25%, 25%
```

Reply:
Including examples like that in the doc comments would help. Maybe also add a link to the relevant MDN page for the way the percentages are used.